sections/intro.tex

```diff
@@ -1,6 +1,290 @@
 \section{Introduction}\label{sec:intro}
 
-This introduction will briefly summarise the literature review for studying a
-patient corpus via clustering. Following this, a condensed data analysis is
-presented highlighting the main conclusions of the clustering and the overall
-benefits compared with traditional condition-treatment segmentation.
+Population health research is becoming increasingly based on data-driven methods
+(as opposed to those designed solely by clinical experts) for patient-centred
+care through the advent of accessible software and a relative abundance of
+electronic data. A vital part of such research is to better understand the
+healthcare needs and behaviours of a population, and it can be beneficial to
+find an appropriate segmentation of that population; such a segmentation allows
+for finer-grained analysis of groups in the population that share some form of
+homogeneity. One commonly used method for such patient-centred analysis is that
+of patient flow and their interaction with the healthcare system.
+
+However, this process relies heavily on detailed data --- about both the system
+and the population within that system --- which may limit research where
+sophisticated data pipelines are not yet in place. This work demonstrates how
+this issue may be overcome using administrative, spell-level hospital data to
```

Comment:
I'm not sure `spell-level` is a sufficiently common term? (I might be wrong)

Reply:
Yes, I'd been thinking that. It's not super common in literature but that's because you'd normally have access to *all* the data. I'll skip over the details here and add more in the data overview section.

Reply:
Sure. It might just need a single sentence saying what is meant by it and even repeating what you've said there (that normally all the data is available).